### PR TITLE
Fixed bug with extra partial being rendered

### PR DESCRIPTION
**Pull request by KevinMulhern**
Fixes https://github.com/TheOdinProject/theodinproject/issues/293

image

**Note**
The style for `alert-none` included in this PR fixes another issue with flashes after registering with omniauth.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class ApplicationController < ActionController::Base
 
   protect_from_forgery
-  include BootstrapFlashHelper
+  # include BootstrapFlashHelper
```
Comment (arun1595) [inline app/controllers/application_controller.rb]: If we don't need this, lets remove the line

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -68,14 +68,12 @@ def store_location
 
   def after_sign_in_path_for(resource)
     if current_user.confirmed_at == nil
-      flash[:partial] = "confirm_email"
+      flash[:warning] = "#{render_to_string :partial => 'layouts/confirm_email'}"
```
Comment (arun1595) [inline app/controllers/application_controller.rb]: Lets go with Ruby 1.9 hash syntax please

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -68,14 +68,12 @@ def store_location
 
   def after_sign_in_path_for(resource)
     if current_user.confirmed_at == nil
```
Comment (arun1595) [inline app/controllers/application_controller.rb]: nit. Can we use `if !current_user.confirmed_at` or `unless current_user.confirmed_at`?

File: app/views/layouts/_confirm_email.html.erb [hunk 4]
```diff
@@ -1,4 +1,2 @@
-<div class="alert alert-warning">
-  <p> Please confirm your email address.  </p> <br>
-  <%= link_to "Didn't receive confirmation instructions, or need them again?", '/confirm_email' %>
-</div>
\ No newline at end of file
+<p> Please confirm your email address.  </p> <br>
+<%= link_to "Didn't receive confirmation instructions, or need them again?", '/confirm_email' %>
```
Comment (arun1595) [inline app/views/layouts/_confirm_email.html.erb]: Please replace the hardwired route with the url helper.

File: app/assets/stylesheets/static_pages.scss [hunk 5]
```diff
@@ -502,6 +502,10 @@
   margin-bottom: 0;
 }
 
+.alert-none {
```
Comment (arun1595) [inline app/assets/stylesheets/static_pages.scss]: Where are we using this class?

Comment (arun1595) [inline app/assets/stylesheets/static_pages.scss]: Oops. Just read your comment. Is there a significance for naming the class `alert-none`?

Comment (KevinMulhern) [inline app/assets/stylesheets/static_pages.scss]: yes thats the I want the name to be none to signify that the flash gets hidden 😄

Comment (arun1595) [inline app/assets/stylesheets/static_pages.scss]: Oh. Fine then :)

**Pull request merged**